.github/workflows/crowdin-ci.yml

```diff
@@ -0,0 +1,68 @@
+name: Crowdin CI
+
+on:
+  schedule:
+    - cron: "20 4 1 * *" # Runs at 4:20 AM on the first day of every month
+  workflow_dispatch: # Can be dispatched manually
+
+jobs:
+  create_approved_language_bucket_prs:
+    runs-on: ubuntu-latest
+    env:
+      CROWDIN_API_KEY: ${{ secrets.CROWDIN_API_KEY }}
+      CROWDIN_PROJECT_ID: ${{ secrets.CROWDIN_PROJECT_ID }}
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+    steps:
+      # Set up environment
+      - name: Check out code
+        uses: actions/checkout@v3
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+
+      - name: Install dependencies
+        run: yarn install
+
+      - name: Install ts-node
+        run: yarn global add ts-node
+
+      - name: Set up git
+        run: |
+          git config --global user.email "actions@github.com"
+          git config --global user.name "GitHub Action"
+
+      - name: Fetch latest dev
+        run: git fetch origin dev
+
+      # Build translations
+      - name: Build Crowdin project
+        id: build-crowdin
+        run: |
+          npx ts-node -O '{"module":"commonjs"}' ./src/scripts/crowdin/translations/triggerBuild.ts;
+          grep BUILD_ID output.env >> $GITHUB_ENV;
```

Comment:
This is the latest recommended way to pass a variable to a future job step. the `set-output` approach originally being used here has been deprecated, so I've converted to this approach.